integration-tests/data/allTestsCases.ts

```diff
@@ -807,7 +796,7 @@ export const commonCases: TestCase[] = [
       ],
     },
   },
-  ...extractOp(0, 151, opMappingProto14).map((op): TestCase => {
+  ...extractOp(0, 154, opMapping).map((op): TestCase => {
```

Comment:
just want to double check is 154 the length of the opMapping?
would it be better to keep this as opMapping.length - 1 in that case?

Reply:
opMapping is actually more than that, this is just the cutoff of what we tested in Lima

Reply:
ahh ok! makes sense to test the ones specific for the protocol!